Makefile.core.mk

```diff
@@ -357,6 +357,25 @@ gen-check: check-no-modify gen check-clean-repo
 sync-configs-from-istiod:
 	cp manifests/charts/istio-control/istio-discovery/files/injection-template.yaml manifests/charts/istiod-remote/files/
 	cp manifests/charts/istio-control/istio-discovery/templates/istiod-injector-configmap.yaml manifests/charts/istiod-remote/templates/
+	cp manifests/charts/istio-control/istio-discovery/templates/configmap.yaml manifests/charts/istiod-remote/templates/
+	cp manifests/charts/istio-control/istio-discovery/templates/configmap-jwks.yaml manifests/charts/istiod-remote/templates/
```

Comment:
This is not needed

Reply:
I'm not sure about configmap-jwks as I haven't tried it... but we do find ConfigMap.yaml is required on remote data plane cluster else istio ingress gw won't reach running.